### PR TITLE
Personal-Portfolio-Website

### PR DESCRIPTION
**Pull request by pallasivasai**
Closes: #726 

- **Title** : Personal-Portfolio-Website
- **Name:** : pallasivasai
- **Idenitfy yourself:** : GSSOC Contributor

### How Has This Been Tested? ⚙️

I tested it locally

### Screenshots 📷

Personal Portfolio - Google Chrome 05-08-2024 10_03_58

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by pallasivasai** [comment]
please tell me what kind of chainages required? @Avdhesh-Varshney

File: Typescript-Projects/Intermediate/Personal-Portfolio-Website/README.md
Comment (Avdhesh-Varshney) [inline Typescript-Projects/Intermediate/Personal-Portfolio-Website/README.md:1]: Follow the readme template

File: Typescript-Projects/README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 |-------|--------------|----------|
 |   1  | [To do list app](./Advanced/To-Do-List-App/)  | ![Advanced](https://img.shields.io/badge/Advanced-FF0000?style=for-the-badge) |
 |   2  | [Quiz App](./Intermediate/Quiz-App/)  | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
+|   3  | [Personal-Portfolio-Website](./Intermediate/Personal-Portfolio-Website/)  | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Typescript-Projects/README.md:30]: Keep it inside the `Basic` category.

Comment (pallasivasai) [reply Typescript-Projects/README.md:30]: Okay done check broh!

**Comment by pallasivasai** [comment]
please check @Avdhesh-Varshney

**Pull request closed without merging**